doc-content/enterprise-only/openshift/template-deploy-namespace-proc.adoc

```diff
@@ -11,14 +11,4 @@ If all image streams were already available in your {OPENSHIFT} environment, you
 
 .Procedure
 
-ifeval::["{context}"=="authoring"]
-. If
-endif::[]
-ifeval::["{context}"!="authoring"]
-If
-endif::[]
-you installed an image streams file according to instructions in <<imagestreams-file-install-proc>>, set the *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`) parameter to the name of your OpenShift project.
-
-ifeval::["{context}"=="authoring"]
-. If you are deploying a high-availability authoring environment and installed an image streams file for AMQ scaledown controller image streams according to instructions in <<imagestreams-file-install-amq-proc>>, set the *AMQ Scaledown Controller ImageStream Namespace* (`AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAMESPACE`) parameter to the name of your OpenShift project.
-endif::[]
+If you installed an image streams file according to instructions in <<imagestreams-file-install-proc>>, set the *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`) parameter to the name of your OpenShift project.
```

Comment:
It throws me off to have a single if structure in a proc with no alternative. I think you could just add the following prereq to this proc:

```
* You installed an image streams file according to the instructions in <<imagestreams-file-install-proc>>.
```

And then make the proc this:

```
Set the *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`) parameter to the name of your OpenShift project.
```

Just a thought.